### ISSUE TITLE
Can't resolve errors

### ISSUE THREAD

**Issue by mcolyer** [stve/airbrake-api#12]
Your readme states that you can resolve errors like so:

```
AirbrakeAPI::Error.update(1696170, :group => { :resolved => true})
```

However I found it necessary to do this in order to get it to resolve:

```
module AirbrakeAPI
  class Error
    def self.resolve(id)
      setup
      put(error_path(id), :body => { :group => {:resolved => true}})
    end
  end
end
```

I think the issue is here (since this isn't a query):
https://github.com/spagalloco/airbrake-api/blob/master/lib/airbrake-api/error.rb#L25

**Comment by stve** [comment]
Thanks for reporting, I'll get a fix out ASAP

**Comment by stve** [comment]
I've updated to use the code that was working for you, thanks for reporting and for the fix as well.  Released in 3.2.1, can you give it a shot, hopefully it's fixed now.  Thanks.
